cassettes/Lead sources controllers/POST/lead-source/saves the number after purchase.json

```diff
@@ -1,54 +0,0 @@
-[
```

Comment:
Just to confirm. You're removing this cassette because you changed the dependency _nock-vcr-recorder-mocha_ to _nock_, right?